src/webgl/material.js

```diff
@@ -1175,6 +1175,84 @@ p5.prototype.shininess = function (shine) {
   return this;
 };
 
+/**
+ * Sets the metalness property of a material used in 3D rendering.
+ *
+ * The metalness property controls the degree to which the material
+ * appears metallic. A higher metalness value makes the material look
+ * more metallic, while a lower value makes it appear less metallic.
+ *
+ * The default and minimum value is 0, indicating a non-metallic appearance.
+ *
+ * Unlike other materials, metals exclusively rely on reflections, particularly
+ * those produced by specular lights (mirrorLike lights). They don't incorporate
+ * diffuse or ambient lighting. Metals use a fill color to influence the overall
+ * color of their reflections. Pick a fill color, and you can easily change the
+ * appearance of the metal surfaces. When no fill color is provided, it defaults
+ * to using white color.
+ *
+ * @method metalness
+ * @param {Number} metallic - The degree of metalness.
+ * @example
+ * <div class="notest">
+ * <code>
+ * let img;
+ * let slider;
+ * function preload() {
+ *   img = loadImage('assets/outdoor_spheremap.jpg');
+ * }
+ * function setup() {
+ *   createCanvas(100, 100, WEBGL);
+ *   slider = createSlider(0, 400, 100, 1);
```

Comment:
Maybe for this slider (and in the next example if we add a slider) we can add a label next to it to say what parameter it's controlling?